pkg/doc/did/doc.go

```diff
@@ -404,6 +410,22 @@ func ParseDocument(data []byte) (*Doc, error) {
 	return doc, nil
 }
 
+func requiresLegacyHandling(raw *rawDoc) bool {
+	if len(raw.PublicKey) == 0 { // docs in v1 format don't have a top-level publicKey array
```

Comment:
|| len(raw.VerificationMethod) > 0

Reply:
|| or &&? if we get a doc that has _no keys_, should we do legacy tweaks?